templates/cluster-template-crs-cni.yaml

```diff
@@ -49,12 +49,17 @@ spec:
       - sysctl --system
       - apt-get -y update
       - DEBIAN_FRONTEND=noninteractive apt-get install -y apt-transport-https curl
-      - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+      - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add
+        -
```

Comment:
Some of the wrapping in this file with the generation updates appear like they might be a bit problematic, have you tried spinning up a cluster with this template? (Running the quickstart e2e test should be sufficient, since it uses this template)